html/elements/iframe.json

```diff
@@ -611,19 +611,19 @@
                 "version_added": false
               },
               "opera": {
-                "version_added": null
+                "version_added": "38"
```

Comment:
51

Same for other flavors.

Reply:
Again, Opera 51? https://www.chromestatus.com/features/5743723954569216 suggests otherwise. I'll change the null...

Reply:
Chrome 51. Opera 38. I'm having an off morning.